### PR TITLE
Support up to v1.18 in deployer containers

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: marketplace/deployer_envsubst_base/Dockerfile [hunk 1]
```diff
@@ -24,12 +24,20 @@ RUN mkdir -p /opt/kubectl/1.14 \
     && chmod 755 /opt/kubectl/1.14/kubectl
 RUN mkdir -p /opt/kubectl/1.15 \
     && wget -q -O /opt/kubectl/1.15/kubectl \
-        https://storage.googleapis.com/kubernetes-release/release/v1.15.5/bin/linux/amd64/kubectl \
+        https://storage.googleapis.com/kubernetes-release/release/v1.15.11/bin/linux/amd64/kubectl \
```
[the remark this answers is not in the document]
Comment (gibbleyg) [reply marketplace/deployer_envsubst_base/Dockerfile]: Done!

File: marketplace/deployer_envsubst_base/Dockerfile [hunk 2]
```diff
@@ -24,12 +24,20 @@ RUN mkdir -p /opt/kubectl/1.14 \
...
     && chmod 755 /opt/kubectl/1.16/kubectl
+RUN mkdir -p /opt/kubectl/1.17 \
+    && wget -q -O /opt/kubectl/1.17/kubectl \
+        https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/linux/amd64/kubectl \
+    && chmod 755 /opt/kubectl/1.17/kubectl
+RUN mkdir -p /opt/kubectl/1.18 \
+    && wget -q -O /opt/kubectl/1.18/kubectl \
+        https://storage.googleapis.com/kubernetes-release/release/v1.18.2/bin/linux/amd64/kubectl \
+    && chmod 755 /opt/kubectl/1.18/kubectl
 RUN ln -s /opt/kubectl/1.14 /opt/kubectl/default
```
[the remark this answers is not in the document]
Comment (gibbleyg) [reply marketplace/deployer_envsubst_base/Dockerfile]: Done!

File: marketplace/deployer_envsubst_base/Dockerfile [hunk 3]
```diff
@@ -14,23 +14,15 @@ RUN pip3 install \
...
-        https://storage.googleapis.com/kubernetes-release/release/v1.15.5/bin/linux/amd64/kubectl \
-    && chmod 755 /opt/kubectl/1.15/kubectl
-RUN mkdir -p /opt/kubectl/1.16 \
-    && wget -q -O /opt/kubectl/1.16/kubectl \
-        https://storage.googleapis.com/kubernetes-release/release/v1.16.2/bin/linux/amd64/kubectl \
-    && chmod 755 /opt/kubectl/1.16/kubectl
-RUN ln -s /opt/kubectl/1.14 /opt/kubectl/default
+RUN for full_version in 1.13.12 1.14.8 1.15.11 1.16.9 1.17.5 1.18.2;  \
+     do \
+        version=${full_version%.*} \
```
Comment (huyhg) [inline marketplace/deployer_envsubst_base/Dockerfile:19]: Nice! I was thinking of having tuples like `"1.13,1.13.12"` and use `cut`.

**Pull request merged**